### PR TITLE
Feat(create-swarmion-app): download version of the example corresponding to the package

### LINKED ISSUE #175: Use versionned templates in `create-swarmion-app`
Currently, `create-swarmion-app` fetches the `examples` folder on its latest version (`main`). However, this is risky since changes can be made on main that have not yet been validated.

In order to fix this issue, we should get the current version in `create-swarmion-app` and use it to build the remote url in create-`packages/swarmion-app/src/create-app.ts`

### PR DESCRIPTION
**Pull request by guillaumeduboc**
resolves #175

### CODE REVIEW

File: packages/create-swarmion-app/src/helpers/examples.ts [hunk 1]
```diff
@@ -28,6 +28,14 @@ const isUrlOk = async (url: string): Promise<boolean> => {
   }
 };
 
+const getFileNameFromBranch = (branch: string): string => {
+  if (branch.match(/v(\d).(\d).(\d)/)) {
```
Comment (fargito) [inline packages/create-swarmion-app/src/helpers/examples.ts:32]: You may encounted issues here with versions that have more than two digits. For example v1.12.488 is a valid version number but will not be matched

Comment (fargito) [reply packages/create-swarmion-app/src/helpers/examples.ts:32]: Also we may want to have alpha or beta releases, how will this be handled ?

Comment (guillaumeduboc) [reply packages/create-swarmion-app/src/helpers/examples.ts:32]: Very good point

Comment (guillaumeduboc) [reply packages/create-swarmion-app/src/helpers/examples.ts]: I added tests to cover these use-cases 😄

**Review by fargito — APPROVED** [review]
Thanks @guillaumeduboc !

**Pull request merged**